virttest/utils_test/qemu/__init__.py

```diff
@@ -698,3 +699,306 @@ def get_all_memorys(self, vm):
         session = self.get_session(vm)
         output = session.cmd_output_safe(cmd, timeout=90)
         return set([_ for _ in output.splitlines() if _])
+
+
+class UsbDevTest(object):
+
+    """
+    Provide basic test for usb devices
+    """
+
+    def __init__(self, test, params, env):
+        self.params = params
+        self.test = test
+        self.env = env
+        self.vm = env.get_vm(params["main_vm"])
+
+    def check_usb_dev_monitor(self, usb_type):
+        """
+        check if the usb device exist in monitor
+        """
+        logging.info("Verify USB device in monitor.")
+        o = self.vm.monitor.info("usb")
+        devnums =[]
+        devs =[]
+        for line in o.splitlines():
+            if "testdev" in line:
```

Comment:
usbdev = qdevices.usbs_define_by_params()
device_id = usbdev.get_aid()
if device_id in monitor.info("usb"):
    return device_id
raise exceptions.TestFail(" USB device '%s' not exists in qtree")